doc/classes/AudioEffectStereoEnhance.xml

```diff
@@ -1,13 +1,17 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="AudioEffectStereoEnhance" inherits="AudioEffect" version="4.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
+		An audio effect that can be used to adjust the intensity of stereo panning.
 	</brief_description>
 	<description>
+		An audio effect that can be used to adjust the intensity of stereo panning.
 	</description>
 	<tutorials>
 	</tutorials>
 	<members>
 		<member name="pan_pullout" type="float" setter="set_pan_pullout" getter="get_pan_pullout" default="1.0">
+			Values greater than 1.0 increase intensity of any panning on audio passing through this effect,
+			whereas values less than 1.0 will decrease the panning intensity. A value of 0.0 will downmix audio to mono.
```

Comment:
Long lines shouldn't be wrapped manually, each line is a separate paragraph in the XML.

Reply:
I pushed an edit with this formatting change.